Ginger/Ginger/BusinessFlowPages_New/NewAutomatePage.xaml.cs

```diff
@@ -972,6 +977,9 @@ private void WorkSpacePropertyChanged(object sender, System.ComponentModel.Prope
                 }
 
                 UpdateToNewSolution();
+
```

Comment:
1- move it inside the null check IF so it will be reset as part of Solution change reset
2- branch has Conflicts- please sync it with latest and solve conflicts